source/components/localisation/Localisation.tsx

```diff
@@ -24,19 +22,24 @@ export default () => {
 		[]
 	)
 
-	const supported = isRegionSupported(localisation)
-
-	const countryName = getCountryNameInFrench(localisation?.country.code)
+	const supportedRegions = useSelector((state) => state.supportedRegions)
+	const isSupported = supportedRegion(localisation?.country?.code)
```

Comment:
nitpick: ici `isSupported` semble être un boolean (du moins son nom le suggère et son utilisation ligne 40 le confirme), or `supportedRegion` ne retourne pas de boolean.

```suggestion
	const isSupported = supportedRegion(localisation?.country?.code) !== undefined
```